src/Sylius/Behat/Service/Resolver/PriceResolver.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Service\Resolver;
+
+final class PriceResolver implements PriceResolverInterface
```

Comment:
Instead of introducing a new class, you should use transformer `LexicalContext` in your steps

Reply:
I've answered on that to Łukasz. Can you look at my answer https://github.com/Sylius/Sylius/pull/13849#discussion_r849441973?